src/globals/scss/_links.scss

```diff
@@ -0,0 +1,29 @@
+@import "import-once";
+@import "colours";
+
+@include exports("links") {
+  a:link {
+    color: $govuk-link-colour;
+  }
+
+  a:visited {
+    color: $govuk-link-visited-colour;
+  }
+
+  a:hover {
+    color: $govuk-link-hover-colour;
+  }
+
+  a:active {
+    color: $govuk-link-active-colour;
+  }
+
+  a {
+    -webkit-tap-highlight-color: rgba($govuk-black, .3);
+  }
+
+  a:focus {
+    outline: 3px solid $govuk-focus-colour;
+    background-color: $govuk-focus-colour;
```

Comment:
No worries! The skiplink styles are here: https://github.com/alphagov/govuk_template/blob/master/source/assets/stylesheets/_accessibility.scss#L70. We can take a look tomorrow.